.github/ISSUE_TEMPLATE/workflows/release.yaml

```diff
@@ -0,0 +1,79 @@
+name: Release workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: "Release version"
+        required: true
+      bundle_channels:
+        description: "Bundle channels"
+        required: true
+        default: "candidates"
+      is_draft_release:
+        description: "Draft release"
+        type: boolean
+        required: false
+        default: true
+      is_pre_release:
+        description: "Pre-release"
+        type: boolean
+        required: false
+        default: false
+
+run-name: Release ${{ inputs.release_version }}
+
+env:
+  GIT_USER: ${{ secrets.GIT_USER }}
+  GITHUB_TOKEN: ${{ secrets.GIT_TOKEN }}
+  VERSION: ${{ inputs.release_version }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Login to quay.io
+      uses: docker/login-action@v3
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USER }}
+        password: ${{ secrets.QUAY_PWD }}
+
+    - uses: actions/checkout@v4
+
+    - name: Build and push operator image
+      run: |
+        make docker-buildx \
+          -e TAG=$VERSION
+
+    - name: Generate bundle metadata
+      run: |
+        make bundle \
+         -e CHANNELS=$CHANNELS
+      env:
+        CHANNELS: ${{ inputs.bundle_channels }}
+
+    - name: Publish bundle in operatorhub.io
+      run: |
+        make bundle-publish \
```

Comment:
We should be able to do the following here to use the username of whoever is triggering the action
``` -e GIT_CONFIG_USER_NAME="${{ github.actor }}"
-e GIT_CONFIG_USER_EMAIL="${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"```

Same below for OpenShift